libs/mosquittoClient/lib/ChangeLog

```diff
@@ -4,11 +4,12 @@ All notable changes to this project will be documented in this file.
 ## [Unreleased]
 
 ## [0.0.1] - 2018-01-02
+## [0.0.2] - 2018-01-18
```

Comment:
Please make new record for your change . For example:
  ## [0.0.2] - 2018-01-18
  ### Added
  <Text ... > [Owner of change]
  
  ### Changed
  Implement onMessage method + little changes [Matus Bodorik]
  
  ### Removed
  None